src/domain/common/whiteboard/excalidraw/collab/utils.ts

```diff
@@ -4,6 +4,11 @@ import {
   InitializedExcalidrawImageElement,
 } from '@alkemio/excalidraw/types/element/types';
 import { unstable_batchedUpdates } from 'react-dom';
+import { FileId } from '@alkemio/excalidraw/types/element/types';
+
+///// This is copied from Excalidraw sources because VITE cannot import certain things from the package,
+///// eventhough VSCode seems to have no problem to see them and shows no compilation error
+// TODO: import them properly
```

Comment:
does it still apply?

Reply:
yes, I'm upgrading @alkemio/excalidraw package now, the new version is packaged with VITE so I'll see if we can import these things from there instead of copy pasting them